contrib/libssh-cmake/linux/x86-64/config.h

```diff
@@ -82,64 +82,37 @@
 /* Define to 1 if you have the <pthread.h> header file. */
 #define HAVE_PTHREAD_H 1
 
-/* Define to 1 if you have eliptic curve cryptography in openssl */
+/* Define to 1 if you have elliptic curve cryptography in openssl */
```

Comment:
For consistency reasons, it would be nice to apply the same changes to the config.h files of the other platforms (--> see contrib/libssh-cmake/*), for example Linux/aarch64, Darwin, etc.